lib/msal-react-native-android-poc/SampleApp/ios/SampleApp.xcodeproj/xcshareddata/xcschemes/SampleApp-tvOS.xcscheme

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same as .plist, may be a good time investment to look into what iOS/Android build files should be .gitignored.